### DISCUSSION TITLE
Deploy model in Cloud, or Huggingface endpoint.

### DISCUSSION THREAD

**Discussion by cmazzoni87** [urchade/GLiNER#161] [Q&A]
Has anyone successfully use the model from a Virtual Machine, or using Vertex AI or Huggingface endpoint instance on the cloud? I been struggling on productionizing Gliner.
Many thanks

**Comment by wjbmattingly** [comment]
> Has anyone successfully use the model from a Virtual Machine, or using Vertex AI or Huggingface endpoint instance on the cloud? I been struggling on productionizing Gliner.
> 
> Many thanks

What if you use gliner-spacy and annotate with style ent. The output would be spaCy ner which HuggingFace can correctly interpret as token classification. You would just load the spaCy pipeline to HuggingFace like a normal spaCy pipeline.

**Comment by cmazzoni87** [comment]
So I managed to create an endpoint using a transformers framework flavor of the model however the Gliner package does not support this. Anyone figured out what is the proper input the model takes for its inference?
